src/common/deploy-geo.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*jshint forin:true, noarg:false, noempty:true, eqeqeq:true, bitwise:true,
+  strict:true, undef:true, curly:false, browser:true,
+  unused:true,
+  indent:2, maxerr:50, devel:true, node:true, boss:true, white:true,
+  globalstrict:true, nomen:false, newcap:true, esnext: true, moz: true  */
+
+/*global require, console, exports, geoip_country_code */
+
+// adapted from:
+
+"use strict";
+
+// bascially, if we think it's august 2015, do a special rollout, otherwise,
+// don't.
+
+const geo = require("./geo");
+const fakeUserCountry = () => Promise.resolve("--");
+
+
+`logic
+
+1.  is it august 2015?  If so, answer proporitional to day
+2.  else:  just answer, darn it
+`
+
+var useReal = function (now, rng) {
+  now = now || new Date();
+  if (rng === undefined) rng = Math.random();
```

Comment:
You're using es6 here, so you could do this more compactly as

```js
var useReal = function (now=new Date(), rng=Math.random()) {
    // ...
}
```

Unlike Python, these defaults will be evaluated for every call, so putting them as default params won't ruin everything.